### DISCUSSION TITLE
how to customize logging in worker

### DISCUSSION THREAD

**Discussion by simonryf** [taskiq-python/taskiq#355] [Q&A]
This is a beginner question and I could not find more info.

I start my worker (separate docker container) (as in [cli - other-parameters](https://taskiq-python.github.io/guide/cli.html#other-parameters)) with:

--no-configure-logging - disables default logging configuration for workers.

but where/how can I then customise my log output?

my broker setup is:

```python
import taskiq_fastapi
from taskiq_nats import NatsBroker
from taskiq_redis import RedisAsyncResultBackend

result_backend = RedisAsyncResultBackend(
    redis_url=str(settings.redis.url.with_path("/1")),
    keep_results=False,
    result_ex_time=900,  # 15min
)
broker = NatsBroker(
    str(settings.nats.url),
    queue="r0bust_queue",
).with_result_backend(result_backend)

taskiq_fastapi.init(
    broker,
    "app.core.application:create",
)
```

which i start in fastapi (lifespan) with:

```python
await broker.startup()
```

Would be glad if someone could point me in the right direction. Thank you.

**Comment by s3rius** [comment]
There's a parameter `--no-configure-logging`  that tells worker to not configure logging itself. After this parameter is used, you can easily setup the logging as you want during the startup.
